LambdaRuntimeDockerfiles/update-dockerfile.ps1

```diff
@@ -0,0 +1,72 @@
+param ([Parameter(Mandatory)]$Dockerfile)
+
+# Updates the Dockerfile with next ASP.NET Core version and checksum256 hash if available
+function Update-Dockerfile ([string]$path) {
```

Comment:
Why does `Update-Dockerfile` name its parameter `$path`  when `$Dockerfile` is used everywhere else?

Reply:
fixed with consistently using DockerfilePath